src/amo/components/AddonMoreInfo/index.js

```diff
@@ -149,8 +172,34 @@ export class AddonMoreInfoBase extends React.Component<InternalProps> {
       );
     }
 
+    const comma = i18n.gettext(',');
+    const renderRelatedCategories = [];
+    const separator = isRTL ? ` ${comma}` : `${comma} `;
+    if (relatedCategories && relatedCategories.length > 0) {
+      relatedCategories.forEach((category, index) => {
+        renderRelatedCategories.push(
+          <Link
```

Comment:
It should probably be an actual list (html). See how it is done for the tags (in another PR linked above).